libpam/pam.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
```

Comment:
Since neither `prefix` nor `exec_prefix` is used in this file, there is no need to define them.

Reply:
From looking at all the pkgconfig files on my computer, it look like it is standard practice to have `prefix` and `exec_prefix` defined in pc files even hen they aren't used.

I removed them anyway as they don't seem to be used by pkgconfig unless you use them explicitly in the files.